### PR TITLE
Adding input functionalities

### PR DESCRIPTION
**Pull request by david0673**
New functionalities:
* retrieve input by id.
* retrieve input by name.
* get input id by name.
* pause/resume input.

These are things that we use in `alooma_utility` that clients can & would want to use.

### CODE REVIEW

File: alooma/alooma.py [hunk 1]
```diff
@@ -128,6 +128,13 @@ def __get_account_name(self):
         res = self.__send_request(requests.get, url)
         return res.json().get('config_clientName')
 
+    def _rest_get_inputs(self):
```
Comment (david0673) [inline alooma/alooma.py]: Do you know why `get_inputs` rely on get_plumbing?
Are the stats & schema field used? those seemed to be the only differences

Comment (david0673) [reply alooma/alooma.py]: & this is much quicker

Comment (yaara1000) [reply alooma/alooma.py]: I think get_plumbing uses the rest endpoint

File: alooma/alooma.py [hunk 2]
```diff
@@ -336,6 +364,15 @@ def edit_input(self, input_post_data):
         res = self.__send_request(requests.put, url, json=input_post_data)
         return res
 
+    def set_input_active_state(self, input_id, pause=False):
```
Comment (yaara1000) [inline alooma/alooma.py]: I'd use a different name since we can either set the input to active or pause it with this function

Comment (david0673) [reply alooma/alooma.py]: I named it like this because i saw it as pause vs resume (not vs active).
How would you like to name it?

**Comment by david0673** [comment]
@yaara1000 any updates here?
I'd like to get this merged :)

**Comment by david0673** [comment]
Thanks @yaara1000, may I merge?

**Pull request closed without merging**